charts/avatar/templates/avatar-api.yaml

```diff
@@ -44,17 +44,4 @@ spec:
             - name: app
               containerPort: 8000
               protocol: TCP
-          resources:
```

Comment:
We don't want to remove these. Its important that we have the ability to configure these on all deployments. I would suggest doing it like tfr-backend where the values file allows you to specify these configurations separately for the api and worker. You already have a separate `worker` section in the values file, you can also create a separate `api` section and define these configurations.